projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts

```diff
@@ -479,6 +482,7 @@ export class IgxDragDirective implements OnInit, OnDestroy {
     public onPointerMove(event) {
```

Comment:
@skrustev Please add the event type.

Reply:
I am using different approaches to getting the pageX/pageY for the different types of interaction, and specifying the types would result in errors from the TypeScript parser.